src/components/Login/index.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+import { Form, Input, Button, Checkbox, Alert } from "antd";
+import { useHistory } from "react-router-dom";
+import firebase from "../../firebaseConfig";
+import { MailOutlined } from "@ant-design/icons";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+
+const layout = {
+  labelCol: { span: 4 },
+  wrapperCol: { span: 8 },
+};
+
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+const SignUp = ({ isTesting }) => {
+  const { t } = useTranslation();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const login = (e) => {
+    e.preventDefault();
+    firebase
+      .auth()
+      .signInWithEmailAndPassword(email, password)
+      .then(() => {
+        setErrorMessage(true);
+      })
+      .catch((err) => {});
+  };
+
+  let history = useHistory();
+  const routeChange = () => {
+    let path = "/forgot";
+    history.push(path);
+  };
+
+  const resetPassword = (e) => {
+    routeChange();
+  };
+
+  return (
+    <Form {...layout} {...tailLayout}>
+      {errorMessage ? (
+        <Alert
+          message="You Can start add your coupon"
+          description="Success"
+          type="success"
+          showIcon
+        />
+      ) : null}
+      {!errorMessage ? <p>{errorMessage}</p> : null}
+      <Form.Item
+        rules={[{ required: true, message: "Please input your username!" }]}
+        value={email}
+        onChange={(e) => setEmail(e.target.value)}
```

Comment:
same thing with forget password 2 onChanges functions I think one of them is extra

Reply:
deleted.